qemu/tests/numa_stress.py

```diff
@@ -138,12 +130,7 @@ def run(test, params, env):
         stress_test.load_stress_tool()
         error_context.context("Get the qemu process memory use status",
                               test.log.info)
-        node_after, memory_after = max_mem_map_node(host_numa_node, qemu_pid)
-        if node_after == most_used_node and memory_after >= memory_used:
-            test.fail("Memory still stick in node %s" % numa_node_malloc)
```

Comment:
Why should this test point be deleted?
The purpose of this patch is to correct the pressure size.
On the other hand, the purpose of this test point is to 
monitor the memory qemu process used should migrate 
from the old node to a new one.

I tested this test point locally and it works as expected.

Reply:
agree that.

Reply:
Discuss with the developers, in fact, Numa balancing functionality depends on host cgroup settings.
On the other hand, the command to put pressure on the host memory through dd cmd in the script cannot achieve the purpose of letting the qemu process release the current host memory and use swap instead.
We have the following two ways to squeeze the host memory to make the qemu process switch to swap, and then release the backlog of memory and transfer it to our expected numa node.

1) Use cgroups to limit qemu process memory usage
2) Extrude by running a pressure program of type Anonymous mapping.

But these have nothing to do with our testing qemu-kvm itself, 
so I agree to remove this test point.